crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -111,6 +112,8 @@ pub struct StandardMaterial {
     pub double_sided: bool,
     /// Whether to cull the "front", "back" or neither side of a mesh
     /// defaults to `Face::Back`
+    // TODO: include this in reflection somehow (maybe via remote types like serde https://serde.rs/remote-derive.html)
```

Comment:
Can we use a blanket impl for Option<T> in bevy_reflect? Or do we not own the Face type either :(